app/views/posts/index.html.erb

```diff
@@ -1,31 +1,31 @@
 <h2>Here is the post for given User</h2>
+  <section class = 'user'>
+    <figure>
+      <image class = 'user-image' src = '<%= @user.photo %>' alt = '<%= @user.name %>'>
+    </figure>
+    <div class='user-info'>
+      <h2>User: <%= @user.name %> </h2>
+      <p>Number of posts: <%= @user.post_counter || 0 %> </p>
```

Comment:
When clicking on the post on the post index page, it should take you to the post-show page. However, this interaction is missing in your case. Kindly fix this issue.

Reply:
Linked